Jenkinsfile

```diff
@@ -94,5 +94,5 @@ def CHANGE_ID = env.CHANGE_ID
 // workaround https://issues.jenkins-ci.org/browse/JENKINS-55987
 // TODO: read this value from Jenkins provided metadata
 def String readCurrentTag() {
-    return sh(returnStdout: true, script: "git describe --tags --match v?*.?*.?* --abbrev=0 --exact-match || echo ''").trim()
+    return sh(returnStdout: true, script: "echo ${TAG_NAME}").trim()
```

Comment:
Nope.

Could you try it?

```suggestion
    return sh(returnStdout: true, script: 'echo ${TAG_NAME}').trim()
```

Reply:
sure... trying right now

Reply:
seems to be working!

Reply:
yep... merging this one to create a new tag and check if problem is solved

Reply:
marking this as solved